drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -39,4 +39,14 @@ config NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE
 	  if the data is larger than the configured size.
 	  Must be a multiple of 4. When set to 0, the feature is disabled.
 
+config NORDIC_QSPI_NOR_XIP
+	bool "XIP (eXecute in place)"
+	depends on SOC_NRF5340_CPUAPP
```

Comment:
Why? This will work on nRF52840 as well.

Reply:
One can avoid such use of interrupts on nRF52840 and still take advantage of XIP.